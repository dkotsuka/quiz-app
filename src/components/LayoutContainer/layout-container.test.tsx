import React from 'react'
import { Text } from 'react-native'
import { LayoutContainer } from '.'
import { create } from 'react-test-renderer';

describe("LayoutContainer", () => {
    
    it("column container matches snapshot", () => {
        const columnContainer = create(<LayoutContainer.Column >
            <Text>First component</Text>
            <Text>Second component</Text>
        </LayoutContainer.Column>)
        expect(columnContainer).toMatchSnapshot()
    })

    it("row container matches snapshot", () => {
        const columnContainer = create(<LayoutContainer.Row >
            <Text>First component</Text>
            <Text>Second component</Text>
        </LayoutContainer.Row>)
        expect(columnContainer).toMatchSnapshot()
    })
})