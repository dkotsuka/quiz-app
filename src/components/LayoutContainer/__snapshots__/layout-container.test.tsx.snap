// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutContainer column container matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "flexDirection": "column",
        "justifyContent": "flex-start",
      },
    ]
  }
>
  <Text>
    First component
  </Text>
  <Text>
    Second component
  </Text>
</View>
`;

exports[`LayoutContainer row container matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
        "justifyContent": "flex-start",
      },
    ]
  }
>
  <Text>
    First component
  </Text>
  <Text>
    Second component
  </Text>
</View>
`;
